cmd/server/main.go

```diff
@@ -201,6 +201,12 @@ func realMain(ctx context.Context) error {
 	processFirewall := middleware.ProcessFirewall(ctx, h, "server")
 	rateLimit := httplimiter.Handle
 
+	// Shared metrics
+	userMetrics, err := user.RegisterMetrics()
```

Comment:
the pattern we've been using is to register these in the controller

https://github.com/google/exposure-notifications-verification-server/blob/main/pkg/controller/issueapi/issueapi.go#L48

I don't think there's a reason to pipe them back out through here.

I think having a shared metrics register w/ a mutex to only initialize once and keep it out of main would be better

Reply:
idk about mutex, but how about  putting it on context